pkg/drivers/kic/oci/network_create.go

```diff
@@ -74,7 +75,13 @@ func CreateNetwork(ociBin string, networkName string) (net.IP, error) {
 	subnetAddr := firstSubnetAddr
 	// Rather than iterate through all of the valid subnets, give up at 20 to avoid a lengthy user delay for something that is unlikely to work.
 	// will be like 192.168.49.0/24 ,...,192.168.239.0/24
-	for attempts < 20 {
+	for attempts < 2 {
```

Comment:
why this was changed from 20 to 2 ?

Reply:
i initially thought to keep the original intention and tryCreateDockerNetwork() two times, each time scanning up to 10 subnets (so trying up to 20 subnets in total), but since the only 'acceptable' (re-triable) error is if network is not free, then we do not need to do that - removed outer loop